addon_toolkit/interfaces/storage.py

```diff
@@ -37,7 +37,7 @@ class StorageConfig:
     external_account_id: str | None = None
 
 
-class ItemType(enum.Enum):
+class ItemType(enum.StrEnum):
```

Comment:
What's the purpose of turning this into StrEnum?

Reply:
This fixes filters for `get_root_items` and `list_child_items`